static/redfish/v1/$metadata/index.xml

```diff
@@ -3691,7 +3756,7 @@
         <edmx:Include Namespace="OemAssembly"/>
         <edmx:Include Namespace="OemAssembly.v1_0_0"/>
     </edmx:Reference>
-    <edmx:Reference Uri="/redfish/v1/schema/OemChassis_v1.xml">
```

Comment:
This looks bad. Could you look into this? We could handle as a separate PR after this merges or as part of this

Reply:
done.